src/app/search-page/searchPage.controller.js

```diff
@@ -3,16 +3,16 @@ import {
 } from '../redux/actions';
 
 export default class SearchPageController {
-    constructor(geoNamesService, routingFunctionsService, $state, $mdDialog, $rootElement, $mdToast, $ngRedux) {
+    constructor(geoNamesService, routingFunctionsService, $state, $mdDialog, $rootElement, $ngRedux, mailingService) {
         this.geoNamesService = geoNamesService;
         this.routingFunctionsService = routingFunctionsService;
         this.nearbyCities = [];
         this.$mdDialog = $mdDialog;
         this.$rootElement = $rootElement;
         this.$state = $state;
-        this.$mdToast = $mdToast;
         this.$ngRedux = $ngRedux;
         this.unsubscribe = this.$ngRedux.connect(this.mapStateToThis)(this);
+        this.mailingService = mailingService;
```

Comment:
it's a good practice maintain a sort criteria here, I mean, first injected services, then variables assignments, calls to functions, etc... and not mess them (e.g. `nearbyCities`)